widget/src/S3FileInput.ts

```diff
@@ -23,83 +23,90 @@ function i18n(text: string): string {
 
 export default class S3FileInput {
   private readonly node: HTMLElement;
+
   private readonly input: HTMLInputElement;
+
   private readonly info: HTMLElement;
+
   private readonly uploadButton: HTMLButtonElement;
+
   private readonly clearButton: HTMLButtonElement;
+
   private readonly spinnerWrapper: HTMLElement;
 
   private readonly baseUrl: string;
+
   private readonly autoUpload: boolean;
+
   private readonly fieldId: string;
 
   constructor(input: HTMLInputElement, options: Partial<S3FileInputOptions> = {}) {
     this.input = input;
 
-    this.baseUrl =
-      options.baseUrl || this.input.dataset.s3fileinput || DEFAULT_BASE_URL;
-    this.autoUpload =
-      options.autoUpload != null
-        ? options.autoUpload
-        : this.input.dataset.autoUpload != null;
+    this.baseUrl = options.baseUrl || this.input.dataset.s3fileinput || DEFAULT_BASE_URL;
+    this.autoUpload = options.autoUpload != null
+      ? options.autoUpload
+      : this.input.dataset.autoUpload != null;
     this.fieldId = this.input.dataset?.fieldId || ''; // TODO this should error out
 
-    this.node = input.ownerDocument!.createElement("div");
-    this.node.classList.add(cssClass("wrapper"));
-    this.node.innerHTML = `<div class="${cssClass("inner")}">
-    <div class="${cssClass("info")}"></div>
-    <button type="button" class="${cssClass("upload")}" disabled>
-      ${i18n("Upload to S3")}
+    this.node = input.ownerDocument.createElement('div');
+    this.node.classList.add(cssClass('wrapper'));
+    this.node.innerHTML = `<div class="${cssClass('inner')}">
+    <div class="${cssClass('info')}"></div>
+    <button type="button" class="${cssClass('upload')}" disabled>
+      ${i18n('Upload to S3')}
     </button>
-    <button type="button" class="${cssClass("clear")}" title="${i18n('Clear (file was already uploaded tho)')}">
-      ${i18n("x")}
+    <button type="button" class="${cssClass('clear')}" title="${i18n('Clear (file was already uploaded tho)')}">
+      ${i18n('x')}
     </button>
-    <div class="${cssClass("spinner-wrapper")}">
+    <div class="${cssClass('spinner-wrapper')}">
       <div class="${cssClass(
-      "spinner"
-    )}"><div></div><div></div><div></div><div></div>
+    'spinner',
+  )}"><div></div><div></div><div></div><div></div>
     </div>
   </div>`;
+    /* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
I think there's a safer way to query all these elements, but since this whole package needs to be significantly refactored, it's not worth trying to address here.